### DISCUSSION TITLE
[Feature] Scan default library

### DISCUSSION THREAD

**Discussion by amrit92** [immich-app/immich#4570] [Feature Request]
### The feature

I think its important to have a scan feature for the existing library. All my photos are in the main library before the external library feature came in. Even if I want to move some photos out, or move everything to an external  library, I cannot do so without rescanning default library.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of the more active #4747
